action.yml

```diff
@@ -22,5 +26,5 @@ outputs:
   reportUrl:
     description: 'Url for the full test execution URL'
 runs:
-  using: 'node12'
+  using: 'node16'
```

Comment:
ya, good call

Reply:
Not a valid option for some reason in GHA :(